### ISSUE TITLE
Docs: make guides for hosting index.html files

### ISSUE THREAD

**Issue by chriskthomas** [chriskthomas/linkfree-generator#16]
It would be nice to make user-friendly docs to help people figure out how to upload the `index.html` files to services like GitHub Pages, Cloudflare Pages, DigitalOcean Web Apps, etc.

**Comment by CharanMN7** [comment]
Hey, I'd like to take the GitHub pages part!

**Comment by chriskthomas** [comment]
Awesome!

**Comment by harsiddhdave44** [comment]
@chriskthomas I don't think it is possible to host index.php on static hosting platforms like GitHub Pages, etc. PHP is rendered at the server side.

**Comment by Adedoyin-Emmanuel** [comment]
@CharanMN7 I would love to work on the user-friendly docs

**Comment by EmekaNkwo** [comment]
Can I make the vercel.md file in the docs folder?

**Comment by mwt** [comment]
> i would like to take the `vercel.md` part

That would be great!
